pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -69,31 +76,30 @@ func (s *Server) initConfigController(args *PilotArgs) error {
 		if err != nil {
 			return err
 		}
+		// Defer starting the controller until after the service is created.
+		s.addStartFunc(func(stop <-chan struct{}) error {
+			go configController.Run(stop)
```

Comment:
don't we have this at the end of the function

Reply:
Fixed.